File: EquipmentApi/EquipmentApi/Data/Repositories/EquipmentModelRepository.cs
using EquipmentApi.Data.Interfaces;
using EquipmentApi.Entities;

namespace EquipmentApi.Data.Repositories
{
    public class EquipmentModelRepository : BaseRepository<EquipmentModel>, IEquipmentModelRepository
    {
        public EquipmentModelRepository(EquipmentDbContext context) : base(context)
        {
        }
    }
}

File: teste-backend-estagio-v2/src/main/java/com/aikoequipment/equipment/repositories/EquipmentModelRepository.java
package com.aikoequipment.equipment.repositories;

import java.util.UUID;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.aikoequipment.equipment.entities.EquipmentModel;

@Repository
public interface EquipmentModelRepository extends JpaRepository<EquipmentModel, UUID> {

}
